app/lib/widgets/cards/productCard/product_card.dart

```diff
@@ -163,3 +162,55 @@ Widget standardFlexProductCard(BuildContext context) {
     ),
   );
 }
+
+@widgetbook.UseCase(
+  name: 'Grid',
+  type: FlexProductCard,
+  path: '[Components]',
+)
+Widget gridFlexProductCard(BuildContext context) {
+  final bool isSaved =
+      context.knobs.boolean(label: 'isProductSaved', initialValue: false);
+
+  return GridView.builder(
+    physics: const AlwaysScrollableScrollPhysics(),
+    gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
```

Comment:
Suggestion: GridView layout overrides FlexProductCard constraints, that's why we've got overflow in the screen in the PR. We should fix that with  `mainAxisExtent` attribute in the gridDelegate to set a size to our cards, no ?

Reply:
Yes. I guess we can add this to our use case so the example isn't broken? 😛  We can also set `aspectRatio` which would solve the issue.

However, in the end, it will be up to the implementation team to correctly size the box according to it's content and where it is being displayed, right?

Reply:
exactly ! 

Yes you right and I'm aligned with this, I think we have to manage the flexibility of our widgets as best we can, but at the end it's the user who has to manage the layout.